libsolidity/AST.cpp

```diff
@@ -324,19 +326,34 @@ void ModifierDefinition::checkTypeRequirements()
 	m_body->checkTypeRequirements();
 }
 
-void ModifierInvocation::checkTypeRequirements()
+void ModifierInvocation::checkTypeRequirements(std::vector<ASTPointer<InheritanceSpecifier>> const& _bases)
```

Comment:
sure. there are some more "std::"'s in this file, should I remove them all?(not from new code)